### DISCUSSION TITLE
Hide image if no URL set

### DISCUSSION THREAD

**Discussion by keshb96** [elementor/elementor#24062] [Developers Community]
Hi

I have an image on my product pages with a custom ACF URL field. This image is used as a size guide button so that when someone clicks on the image, the size guide image pops up.

However, some products do not have a size guide. Can I set the product page to hide this image if there is no size guide URL?

Thanks in advance

![Edit-product-“Men’s-Derby-Tweed-Leighton-Suit-Trousers”-‹-Walker-Hawkes-—-WordPress](https://github.com/elementor/elementor/assets/148566018/b1f6604a-89f4-4cff-9f8d-c4ace84a3806)
![Edit-Single-Product-with-Elementor](https://github.com/elementor/elementor/assets/148566018/f429513e-4c06-433d-b21e-55c7396a9985)

**Comment by perks98** [comment]
You can try this free dynamic conditions plugin to hide that image widget if value is empty : https://wordpress.org/plugins/dynamicconditions/

**Reply by keshb96** [reply]
Thank you!
